core/src/test/scala/integration/kafka/server/QuorumTestHarness.scala

```diff
@@ -340,6 +340,7 @@ abstract class QuorumTestHarness extends Logging {
 
 object QuorumTestHarness {
   val ZkClientEventThreadSuffix = "-EventThread"
+  val RaftClientThreadPrefix = "raft"
```

Comment:
Was there a plan to use this?

Reply:
I think I extracted this from another PR where I made a more substantial change. I'll figure out what I had in mind.